native/jni/magiskpolicy/rules.cpp

```diff
@@ -176,6 +176,10 @@ void sepolicy::magisk_rules() {
     // Allow Zygisk to prctl PR_SET_MM
     allow("zygote", "zygote", "capability", "sys_resource");
 
+    // Allow hook
+    allow("zygote", "zygote", "process", "execmem");
+    allow("system_server", "system_server", "process", "execmem");
```

Comment:
This is not needed.

Reply:
appdomain allowed, zygote allowed, why not also let system_server allow execmem? system_server is a common hook target.

Reply:
`process execmem` SELinux is used for JIT

I know this because GrapheneOS [`disables theses rules`](https://github.com/GrapheneOS/platform_system_sepolicy/commit/680e6bcc1b0d1e5438e3467b54da132261c4f0b3) because they only allow AOT and not JIT

But modules or Zygisk itself might be dependent on that if they want to load custom code.